ui/pages/login.tsx

```diff
@@ -48,7 +48,10 @@ const Login: NextPage = () => {
       kratos
         .getLoginFlow({ id: String(flowId) })
         .then((res) => setFlow(res.data))
-        .catch(handleFlowError("login", setFlow));
+        .catch(handleFlowError("login", setFlow))
+        .catch(() => {
+          window.location.href = "./oidc_error";
```

Comment:
question: I don't know about the latest changes in the login UI, but I thought that this page was used only to display errors coming from Hydra (related to the OIDC flow). I would think that we would redirect the user to `error` instead, am I missing something?

Reply:
Agreed, this should be a dedicated error message and not using the oidc error page. Will change this shortly.